components/global/Header/_subsubnavigation.scss

```diff
@@ -0,0 +1,108 @@
+@use "abstracts/functions";
+
+$_color: functions.sass-palette(neutral80);
+$_bg-color: #E8E8E8;
+$_button-bg-color: functions.sass-palette(neutral02);
+
+.c-sub-subnav-list {
+  height: 100%;
+  max-height: 0;
+  margin-left: functions.header(nav-link-padding-lateral) / 2;
+  overflow: auto;
+  color: $_color;
+  visibility: hidden;
+  transition: max-height 0.25s ease-out, visibility 0s 0.25s;
+
+  &--is-active {
+    max-height: calc(100vh - var(--header-height));
+    visibility: visible;
+    transition: max-height 0.5s ease-out;
+  }
+
+  &__item {
+    position: relative;
+
+    &:first-child::after {
+      position: absolute;
+      bottom: 0;
+      left: functions.header(nav-link-padding-lateral);
+      display: block;
+      width: calc(100% - #{2 * functions.header(nav-link-padding-lateral)});
+      height: 1px;
+      content: "";
+      background-color: currentcolor;
+
+      .c-nav-list--mobile & {
+        left: functions.header(nav-link-padding-lateral) / 2;
+        width: calc(100% - #{2 * functions.header(nav-link-padding-lateral) / 2});
+      }
+    }
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    text-decoration: none;
+    background-color: $_bg-color;
+    transition: color 0.2s, background-color 0.2s;
+
+    &:hover,
+    &:focus-visible,
+    &--is-active {
+      color: functions.palette(white);
+      background-color: functions.palette(turquoise50);
+    }
+
+    .c-subnav-list__item & {
+      justify-content: space-between;
+      flex-direction: row-reverse;
+    }
+
+    .c-sub-subnav-list__item & {
+      justify-content: space-between;
+      flex-direction: row;
+    }
+
+    .c-nav-list--mobile & {
+      height: functions.header(height);
+      padding: functions.header(nav-link-padding-lateral) / 3 functions.header(nav-link-padding-lateral) / 2;
+    }
+
+    .c-nav-list--desktop & {
+      height: functions.header(height);
+      padding: functions.header(nav-link-padding-lateral);
+    }
+
+    .c-sub-subnav-list__item-inner > & {
+      color: #333;
+      background-color: $_button-bg-color;
+
+      &:hover,
+      &:focus-visible,
+      &--is-active {
+        color: functions.palette(white);
+        background-color: functions.palette(turquoise50);
+      }
+
+      .c-nav-list--mobile & {
+        padding: functions.header(nav-link-padding-lateral) / 2 functions.header(nav-link-padding-lateral);
+      }
+    }
+
+    &-icon {
+      width: 24px;
+      height: 24px;
+      transform: rotate(-90deg);
+    }
+
+    &-text {
+      max-width: calc(100% - 34px);
+      text-align: left;
+    }
+
+    &--is-active &-icon {
+      transform: rotate(90deg);
```

Comment:
I don't love rotating an icon just to get the starting orientation I need, but there was not caret with the correct orientation, and I was always gonna rotate it onclick anyhow so 🤷